helper.py

```diff
@@ -12,7 +12,7 @@
 FORMATTER = logging.Formatter(
     "%(asctime)s:%(name)s:%(module)s:%(levelname)s:%(message)s"
     )
-LOG_FILE = "var/log/ansible-run-monitoring.log"
+LOG_FILE = "var/log/monitoring/ansible-run-monitoring.log"
```

Comment:
this should probably be in the config file, you can add it as `ANSIBLE_MONITORING_LOG_FILE` or something if it conflicts with anything else in the config (since we're wanting to use the same one for the DNAnexus run monitoring)

Reply:
Ignore me this is in a docker container, my monday brain no work

Reply:
Valid point tho, if there's a chance that the logging directory might change in the future, then a flexible logging directory in config txt might make sense